### PR TITLE
Revert "update 64-bit check wording"

### PR DESCRIPTION
**Pull request by come-nc**
This reverts #34906
See https://github.com/nextcloud/server/issues/36072

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: core/js/setupchecks.js [hunk 1]
```diff
@@ -421,7 +421,7 @@
 						messages.push({
 							msg: t(
 								'core',
-								'It seems like you are running a 32-bit PHP version. Nextcloud 26 and higher require 64-bit. Please upgrade your OS and PHP to 64-bit! For further details read {linkstart}the documentation page ↗{linkend} about this.'
+								'It seems like you are running a 32-bit PHP version. Nextcloud needs 64-bit to run well. Please upgrade your OS and PHP to 64-bit! For further details read {linkstart}the documentation page ↗{linkend} about this.'
```
Comment (szaimen) [inline core/js/setupchecks.js:424]: Question: should we remove the complete warning or does it still make sense to show this?

Comment (come-nc) [reply core/js/setupchecks.js:424]: I think it should stay, 32bit installations cannot handle dates before 1970 or after 2038, and may still have some trouble with large files.

Comment (szaimen) [reply core/js/setupchecks.js:424]: Okay good then

**Review by szaimen — APPROVED** [review]
LGTM

**Review by blizzz — APPROVED** [review]

**Pull request merged**